students/paul_c/session04/kata_fourteen.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+
+import sys
+import random
+
+"""
+This program was tested using the sherlock_small.txt and a_case_of_identity.txt files located in the working directory.
+
+"""
+
+def build_text(word_pairs):
+        """
```

Comment:
You doubled the indentation within this function (8 spaces instead of 4). Not a big issue, but in general, the indentation should be consistent across your code.